src/wallet/rpcwallet.cpp

```diff
@@ -3222,8 +3222,6 @@ Value z_sendmany(const Array& params, bool fHelp)
             throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid parameter, expected object");
         const Object& o = output.get_obj();
 
-        RPCTypeCheck(o, boost::assign::map_list_of("address", str_type)("amount", real_type));
```

Comment:
Shouldn't this be replaced by a type check that accepts a real or integer. How does upstream type-check similar cases?